eng/versioning/version_data.txt

```diff
@@ -34,7 +34,7 @@ com.microsoft.azure:azure-keyvault-cryptography;1.2.4;1.3.0-beta.1
 com.microsoft.azure:azure-keyvault-extensions;1.2.4;1.3.0-beta.1
 com.microsoft.azure:azure-keyvault-test;1.2.3;1.2.4
 com.microsoft.azure:azure-keyvault-webkey;1.2.4;1.3.0-beta.1
-com.microsoft.azure:azure-servicebus;3.6.2;3.7.0-beta.1
+com.microsoft.azure:azure-servicebus;3.6.2;3.6.3
```

Comment:
Is the plan to ship this once this is merged?

Reply:
Yes.